src/components/Dialogs/DialogItem/DialogItem.jsx

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
+import { NavLink } from 'react-router-dom';
+
 import styles from './../Dialogs.module.css';
-import { NavLink } from "react-router-dom";
 
 const DialogItem = (props) => {
-    let path = "/dialogs/" + props.id;
+	let path = "/dialogs/" + props.id;

```

Comment:
did you setup eslint locally? it should have warned you that you should not use let variables
```suggestion
	const path = `/dialogs/${props.id}";

```

Reply:
also, prefer using interpolation instead of String "" +, just like suggested above